cellprofiler/gui/pathlist.py

```diff
@@ -114,7 +114,7 @@ def __init__(self, *args, **kwargs):
             self.SetFont(self.DROP_FILES_AND_FOLDERS_FONT)
             self.drop_files_and_folders_text_extent = self.GetFullTextExtent(
                 self.DROP_FILES_AND_FOLDERS_HERE
-            )
+            )[:2]
```

Comment:
Did the return type for this change? This seems suspect

Reply:
tbh, i'm not entirely sure how to interpret the documentation at https://wxpython.org/Phoenix/docs/html/wx.GraphicsContext.html#wx.GraphicsContext.GetFullTextExtent but after stepping through the code, it returned 4 values, hence the splice. Please let me know if a better method comes to mind.